fractions/cdi-extensions/cdi-jaxrsapi/src/main/java/org/wildfly/swarm/cdi/jaxrsapi/runtime/ClientServiceFactory.java

```diff
@@ -0,0 +1,166 @@
+package org.wildfly.swarm.cdi.jaxrsapi.runtime;
```

Comment:
fwiw, the changed I slipped into the fraction-plugin (v55, released, used here) works a lot harder to ensure that when we add :deployment modules, and they add :main, that other things don't slip into the actual deployment module, making CDI grumpy.

I had an issue where jaxrsapi's :deployment module was mixed into the user's .war, which then depended on the jaxrsapi:main module (which is currect), but then Weld saw the `META-INF/beans.xml` in that module, found the jar, and then CDI'd the whole dang jar, including both the `*Fraction` and everything in its `runtime.*` package.  And then complained. 

I think it was complaining more quietly before, but this PR's changes resulted in more than a slight whinge, and caused full-blown errors.

So, now, :deployment modules depend on :main, as before, but explicitly exclude `META-INF` in that dependency, to avoid Weld going far afield to find components.